### PR TITLE
Point cloud acquired frame

### PR DESCRIPTION
**Pull request by dNechita**

### CODE REVIEW

**Review by SeptimiuVana — APPROVED** [review]

File: bindings/python/examples/showPointCloud/showPointCloud.py [hunk 1]
```diff
@@ -52,134 +53,201 @@ def transform_image(np_image):
...
+        status = cameras[0].initialize(config)
+        if not status:
+            print("cameras[0].initialize() failed with status: ", status)
+
+        types = []
+        status = cameras[0].getAvailableFrameTypes(types)
+        if not status:
+            print("system.getAvailableFrameTypes() failed with status: ", status)
+
+        status = cameras[0].setFrameType("lr-qnative")
```
Comment (VHolonec) [inline bindings/python/examples/showPointCloud/showPointCloud.py:88]: Now on dev-5.1.0 branch this method has another name like setMode

File: bindings/python/examples/showPointCloud/showPointCloud.py [hunk 2]
```diff
@@ -52,134 +53,201 @@ def transform_image(np_image):
...
+        camera1 = cameras[0]
+
+        status = cameras[0].initialize(config)
+        if not status:
+            print("cameras[0].initialize() failed with status: ", status)
+
+        types = []
+        status = cameras[0].getAvailableFrameTypes(types)
+        if not status:
+            print("system.getAvailableFrameTypes() failed with status: ", status)
```
Comment (VHolonec) [inline bindings/python/examples/showPointCloud/showPointCloud.py:86]: Now on dev-5.1.0 branch this method has another name like getAvailableModes

**Comment by dNechita** [comment]
> In January we removed this example (#556). It makes sense to keep only the fix.

Looks like we removed the C++ example and not the Python example. So we will keep this.

File: bindings/python/examples/showPointCloud/showPointCloud.py (hunk 2 above)
Comment (liealex) [reply bindings/python/examples/showPointCloud/showPointCloud.py]: Solved

File: bindings/python/examples/showPointCloud/showPointCloud.py (hunk 1 above)
Comment (liealex) [reply bindings/python/examples/showPointCloud/showPointCloud.py]: Solved

**Pull request merged**